### ISSUE TITLE
Github pages

### ISSUE THREAD

**Issue by dhzdhd** [dhzdhd/Obsidian#13]
Implement documentation for every command in gh-pages

**Comment by Arnav-2004** [comment]
Hi, @dhzdhd do you have any general layout in mind?

**Comment by dhzdhd** [comment]
> Hi, @dhzdhd do you have any general layout in mind?

Not really. You can make a demo of sorts if you want and we can figure out the layout on the go.

**Comment by ItzLukaDev** [comment]
I would be interested in working on this.

**Comment by dhzdhd** [comment]
@Arnav-2004 work on the content for documentation for the bot.
@ItzLukaDev work on the tailwind css and overall layout and integration with jekyll.

**Comment by ItzLukaDev** [comment]
when Can I pr the bot website?

**Comment by ItzLukaDev** [comment]
Update: I have upgraded to tailwindcss cdn version 3.

This gives some nice features like gradients and dark mode, so I'll be posting some screenshots of the new  website here when I am done.

**Comment by dhzdhd** [comment]
> Update: I have upgraded to tailwindcss cdn version 3.
> 
> This gives some nice features like gradients and dark mode, so I'll be posting some screenshots of the new website here when I am done.

Make a PR, committing all your work and post images there.
Regarding the CDN way of using tailwind, it's fine for now.

**Comment by ItzLukaDev** [comment]
I'll be free for another week, so we can improve the text and href's too soon. We might want to migrate to tailwind installed via npm, since that does give intellisense support so we can fish out bugs and unecessary classes faster.

**Comment by dhzdhd** [comment]
> I'll be free for another week, so we can improve the text and href's too soon. We might want to migrate to tailwind installed via npm, since that does give intellisense support so we can fish out bugs and unecessary classes faster.

I will look into the tailwind package.
As for the text - it is fine for now although I welcome any improvements to it

**Comment by Akulbcodes** [comment]
Heyo I have made the basic ui for the docs page

![image](https://user-images.githubusercontent.com/101550098/163700837-a57cb11e-9de8-4bc1-a528-0ae37a8f3564.png)
![image](https://user-images.githubusercontent.com/101550098/163700841-4906edf1-dd2d-4872-bdb3-99eaea3a13d8.png)

**Comment by dhzdhd** [comment]
Make a PR
